src/app/dataset-view/dataset.subscriptions.service.ts

```diff
@@ -55,6 +56,14 @@ export class DatasetSubscriptionsService {
         return this.sqlError$.asObservable();
     }
 
+    public emitInvolvedSqlDatasetsId(ids: string[]): void {
+        this.involvedSqlDatasetsId$.next(ids);
+    }
+
+    public get involvedSqlDatasetsIdChanges(): Observable<string[]> {
+        return this.involvedSqlDatasetsId$.asObservable();
+    }
+
```

Comment:
I don't see a point why do we split the notification with a ready query result on a few observables. They always come together. 

Such splits made sense for other dataset parts, where we had a main query, and it's parts have been used in multiple tabs. But this is rather not the case for SQL query results.